packages/cache-memcached/package.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
todo(this-file): I believe we are missing a couple of dependencies:
- `@medusajs/types` should be a `devDependency`
- `@medusajs/modules-sdk` should be a `dependency`

Also, why do we need a dependency on `@medusajs/medusa`?

Reply:
Good catch, we don't need it as a dep anymore after rebase.